server/runtime/src/main/java/org/infinispan/server/logging/Messages.java

```diff
@@ -54,11 +54,11 @@
    @Message("Specifies a logging configuration file. Defaults to `%s`.")
    String serverHelpLoggingConfig(String defaultConfiguration);
 
-   @Message("Sets the name of the cluster. Defaults to `%s`.")
-   String serverHelpClusterName(String defaultClusterName);
+   @Message("Sets the name of the cluster. Default set by configuration expression")
+   String serverHelpClusterName();
```

Comment:
I disagree with this change. If you remove the expression, it will default to `cluster`.

Reply:
Actually, ignore me: after looking at https://github.com/infinispan/infinispan/pull/8782 I think we should rely exclusively on the property substitution without pushing defaults into the sysprops for both cluster and stack